config/app.json

```
@@ -6,6 +6,9 @@
     "inline",
     "column-2"
   ],
+  "events":{
+    "userStatus":["Attending","Speaking"]
```

Looking at the usage code from here, it is still not readable or maintainable

```js
event.userStatus == config.events.userStatus[1]
```

So I suggest using an object not an array

```suggestion
    "userStatus": { attending: "Attending", speaking: "Speaking"}
```

Then the usage will become

```js
event.userStatus == config.events.speaking
```